spring-core/src/main/java/org/springframework/core/io/AbstractResource.java

```diff
@@ -163,14 +162,6 @@ public long contentLength() throws IOException {
 			}
 			return size;
 		}
-		finally {
```

Comment:
Please revert. The exception thrown while closing the stream is not caught/logged anymore.
See [this message](https://mail.openjdk.org/pipermail/coin-dev/2009-April/001503.html) for a discussion on possible workarounds. I think we should keep the current code.